tensorflow/core/common_runtime/mkl_cpu_allocator.h

```diff
@@ -21,7 +21,6 @@ limitations under the License.
 
 #ifdef INTEL_MKL
 
-#include <unistd.h>
```

Comment:
should we wrap this with "ifndef _Win32" instead?

Reply:
Hi @gunan , Since unistd.h is not included in the master branch and we don't need it either, I kept this deleted.